src/Controller/EnMarche/TerritorialCouncil/PoliticalCommitteeController.php

```diff
@@ -116,7 +116,7 @@ public function listOfficialReportsAction(
     }
 
     /**
-     * @Route("/proces-verbaux/{uuid}/telecharger", name="official_report_download", methods={"GET"})
+     * @Route("/proces-verbaux/{uuid}", name="official_report_download", methods={"GET"})
```

Comment:
do we need to change the page title too?

Reply:
Which title?